litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowGraphs/ScheduleAndRunStats/style.ts

```diff
@@ -0,0 +1,16 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  workflowGraphs: {
+    padding: theme.spacing(3.125, 3.75, 3.125, 10),
+    height: '100%',
+    marginBottom: theme.spacing(3.125),
+  },
+  graphContainer: {
+    width: '900px',
+    height: '320px',
+    marginLeft: '-70px',
+    marginBottom: '25px',
```

Comment:
rem, margin in theme and use one margin property

Reply:
I think it's a bug from components, already discussed with @whitetiger1399 , it's not taking theme.spacing()

Reply:
no issues, try to move the px to rem

Reply:
Yes `rem` should work